### ISSUE TITLE
Icon missing

### ISSUE THREAD

**Issue by ykelvis** [alexanderjeurissen/ranger_devicons#94]
- [x] I've read the [guidelines for Contributing](https://github.com/alexanderjeurissen/ranger_devicons/blob/main/CONTRIBUTING.MD)

**Describe the bug**
ranger_devicons is not loaded

**To Reproduce**
1. install ranger_devicons properly
2. launch ranger

**Environment (please complete the following information):**
 - archlinux 5.14.2-arch1-2
 - zsh 5.8 (x86_64-pc-linux-gnu)

**Ranger devicons (please complete the following information):**
 - version / commit [e.g. HEAD]
 - output of `tree ~/.config/ranger` [can be installed on most OS using default package manager e.g. `apt-get install tree`]
```
/home/user/.config/ranger/
├── commands_full.py
├── commands.py
├── plugins
│   ├── __init__.py
│   ├── __pycache__
│   │   └── __init__.cpython-39.opt-1.pyc
│   └── ranger_devicons
│       ├── CONTRIBUTING.MD
│       ├── devicons.py
│       ├── __init__.py
│       ├── LICENSE
│       ├── LICENSE_NERDFONT
│       ├── __pycache__
│       │   ├── devicons.cpython-39.opt-1.pyc
│       │   └── __init__.cpython-39.opt-1.pyc
│       ├── README.md
│       └── screenshot.png
├── rc.conf
├── rifle.conf
└── scope.sh
```

**Additional context**
#92 uses env to set these XDG_DIRs, but dit not provide a default path. if those variables are not set (when using a wm not de), it would cause an `index out of range` error. by comparison, icons are showing when using f3be2e69993514ec78e210af2efc3759847134c8

`ranger -d` output

```
ranger version: ranger 1.9.3
Python version: 3.9.7 (default, Aug 31 2021, 13:28:12) [GCC 11.1.0]
Locale: en_US.UTF-8

Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ranger/core/main.py", line 447, in load_settings
    module = importlib.import_module('plugins.' + plugin)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/user/.config/ranger/plugins/ranger_devicons/__init__.py", line 4, in <module>
    from .devicons import *
  File "/home/user/.config/ranger/plugins/ranger_devicons/devicons.py", line 12, in <module>
    XDG_DOCUMENTS_DIR = XDG_DOCUMENTS_DIR.split('/')[-2]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ranger/core/main.py", line 119, in main
    load_settings(fm, args.clean)
  File "/usr/lib/python3.9/site-packages/ranger/core/main.py", line 454, in load_settings
    fm.notify(ex_msg, bad=True)
  File "/usr/lib/python3.9/site-packages/ranger/core/actions.py", line 177, in notify
    raise Exception(str(obj))
Exception: Error while loading plugin 'ranger_devicons'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

**Comment by Head0nF1re** [comment]
See my issue #93

**Comment by ykelvis** [comment]
fixed by #95 , thanks for the help
